clients/js/package.json

```diff
@@ -35,35 +35,30 @@
     "format": "prettier --cache --write . && eslint --ignore-path .gitignore --fix .",
     "build": "npm run build:esm && npm run build:cjs",
     "clean": "rm -rf lib",
-    "build:esm": "tsc -p ./tsconfig.json",
+    "build:esm": "tsc -p ./tsconfig.esm.json",
     "build:cjs": "tsc -p ./tsconfig.cjs.json && node scripts/rename-cjs",
-    "test": "jest",
-    "coverage": "jest --coverage",
+    "test": "jest --coverage",
     "prepublishOnly": "npm run build",
-    "proxy": "node --loader ts-node/esm  scripts/proxy.ts"
+    "proxy": "node --loader ts-node/esm scripts/proxy.ts"
   },
   "dependencies": {
-    "@oasisprotocol/deoxysii": "0.0.5",
-    "cborg": "1.10.2",
-    "ethers": "6.10.0",
     "@noble/hashes": "1.3.2",
-    "tweetnacl": "1.0.3",
-    "type-fest": "2.19.0"
+    "@oasisprotocol/deoxysii": "0.0.6",
+    "cborg": "1.10.2"
   },
   "devDependencies": {
     "@types/jest": "^29.5.11",
     "@types/node": "^18.7.18",
-    "@types/node-fetch": "^2.6.2",
     "@typescript-eslint/eslint-plugin": "^5.38.0",
     "@typescript-eslint/parser": "^5.38.0",
     "eslint": "^8.23.1",
     "eslint-config-prettier": "^8.5.0",
+    "ethers": "6.x",
```

Comment:
Leftover?

Reply:
Yup it's in dev dependencies, it's used for various tests & proxy server stuff. It doesn't affect the released package.